org.opentosca.container.core/src/main/java/org/opentosca/container/core/next/services/instances/PlanInstanceService.java

```diff
@@ -85,6 +85,14 @@ public void deletePlanInstances(final Csar csar) {
             .collect(Collectors.toList());
     }
 
+    public PlanInstance getPlanInstanceByIdWithConnectedEntities(final long id) {
```

Comment:
Why do you call it "withConnectedEntities" ? It sounds to me a little like "with connected nodes"

Reply:
Do you have a better idea? WithInitializedAttributes? I thought we load everything with the entity graph —> withConnectedEntities. But I`m open for wording improvements :D